new/LIT6166MiraclesJohnBaptist.xml

```diff
@@ -53,10 +55,170 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     </teiHeader>
     <text>
         <body>
+          <div type="edition">
+            <note>The text follows <ref type="mss" corresp="ESmy023"/>.</note>
+             <div type="textpart" subtype="chapter" xml:id="Introduction">
+                 <label>Introduction</label>
+                 <ab>
+                   <pb n="78ra"></pb>
+                   በሰመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡
+                   ንዌጥን፡ በረድኤት፡ እግዚአብHኤር፡ ጽሒፈ፡ ተአምራቲሁ፡ ወመንክራቲሁ፡ ለኮከበ፡ ገዳመ፡
+                   ዮርዳኖስ።
+                 </ab>
+              </div>
+
+              <div type="textpart" subtype="chapter" xml:id="Miracle1">
+                 <label>Miracle 1</label>
+                 <ab>
+                   <pb n="78rb"></pb>
+                   በ፩ተአምረሁ፡ ለብፁዕ፡ ነቢይ፡
+                   ወሰማዕት፡ ቅዱስ፡ ዮሐንስ፡
+                    መጥምቅ፡ ዘተኃርየ፡ እምከርሠ፡
+                    እሙ፡ ከመ፡ ኤርምያስ፡ ነቢዩ። ስምዑ፡ ንነግረክሙ፡ አኃዊነ፡ አድ፡ ወአንስት፡ <gap reason="omitted" resp="DN"/>
+                    ወ<pb n="78va"></pb>ነበረ፡ ፩ብእሲ፡ ዘረሰየ፡ ብሔሮ፡ መልዕልተ፡ አሐቲ፡ ደብር፡ ዘምድረ፡ ኅርጉድ፡
+                 </ab>
+              </div>
+
+              <div type="textpart" subtype="chapter" xml:id="Miracle2">
+                <label>Miracle 2</label>
+                <ab>
+                  <pb n="79rb"></pb>
+                  ፪ተአምረሁ፡ ለብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ መጥምቅ፡ ጸሎቱ፡ የሃሉ፡ ምስለ፡ አግብርቲሁ፡
+                  አቡነ፡ ወልደ፡ ኪዳን፡ ለዓለመ፡ ዓለም፡ አሜን።
+                  ወካዕበ፡ ስምዑ፡ ንንግርክሙ፡ ዘገብረ፡ ሎቱ፡ ተአምረ፡ ለውእቱ፡ ብእሲ፡ ዘአቅደምን፡ ነጊረ፡
+                  በእንቲአሁ። <gap reason="omitted" resp="DN"/> ወሀለውዎ፡ ሰብአ፡ ደም፡
+                  ፅኑዓነ፡ ኃይል፡ እለ፡ ርኁቅ፡ ብሔሮሙ፡
+                </ab>
+              </div>
+
+
+              <div type="textpart" subtype="chapter" xml:id="Miracle3">
+                 <label>Miracle 3</label>
+                 <ab>
+                   <pb n="79vb"></pb>
+                   ፫ተአምረሁ፡ ለብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ መጥምቅ፡ ጸሎቱ፡ ወበረከቱ፡ የሃሉ፡ አቡነ፡ ዕቁባ፡ ዮሐንስ፡ ወለተ፡ ኢየሱስ፡ ለዓለመ፡ ዓለም፡ አሜን።
+                   ስምዑ፡ ካዕበ፡ ዘገብረ፡ ተአምረ፡ ወመንክረ፡ ቅዱስ፡ ዮሐንስ፡ መጥምቅ፡ ዘበእዴሁ፡ ገሠሠ፡ ርእሶ፡ ለመለኮት፡ በውስተ፡ ዮርዳኖስ፡
+                   <gap reason="omitted" resp="DN"/> <pb n="80ra"></pb>
+                   ወሀለዉ፡ ሰብእ፡ ቅሩባን፡ እለ፡ ይነብሩ፡ ደወለ፡ ሀገሩ፡ ለይኩኖ፡ እግዚእ፡ እለ፡ ይጸልዕዎ፡ በአልባሲሆሙ፡
+
+                 </ab>
+              </div>
+              <div type="textpart" subtype="chapter" xml:id="Miracle4">
+                <label>Miracle 4</label>
+                <ab>
+                  <pb n="80va"></pb>
+                  ፬ተአምረሁ፡ ለቅዱስ፡ ዮሐንስ፡ ጸሎቱ፡ ወበረከቱ፡ የሃሉ፡ ምስለ፡ አግብርቲሁ፡ ወልደ፡ ኪዳን፡
+                  ወለተ፡ ገብ<supplied reason="omitted">ር</supplied>ኤል፡ ለ<supplied reason="omitted">ዓለመ</supplied>፡
+                  ዓ<supplied reason="omitted">ለም</supplied>፡ አ<supplied reason="omitted">ሜን</supplied>።
```

Comment:
not so relevant for the work record, but maybe for the manuscript record - maybe this could be marked up as an abbreviation, not as omitted?

Reply:
indeed. thanks.